lib/active_storage/service/postgresql_service.rb

```diff
@@ -103,7 +103,7 @@ def generate_url(key, expires_in:, filename:, disposition:, content_type:)
         )
 
         generated_url = url_helpers.rails_postgresql_service_url(verified_key_with_expiration,
-          **url_options,
+          only_path: true,
```

Comment:
Let's add a comment about this:

```suggestion
          only_path: true,  # This fixes an exception with attachment URL generation from a worker: https://github.com/AllYourBot/hostedgpt/pull/398#issuecomment-2168135853
```